airflow/config_templates/config.yml.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string"
+      },
+      "description": {
+        "type": ["string", "null"]
+      },
+      "options": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "description": {
+              "type": ["string", "null"]
+            },
+            "version_added": {
+              "type": ["string", "null"]
+            },
+            "type": {
+              "type": "string",
+              "enum": ["string", "boolean", "integer", "float"]
+            },
+            "example": {
+              "type": ["string", "null", "number"]
+            },
+            "default": {
+              "type": ["string", "null", "number"]
+            },
+            "sensitive": {
```

Comment:
Can you add description to this field?  It is not obvious when this field should be set.